ci/cppcheck/cppcheck-new-issues.sh

```diff
@@ -25,8 +25,19 @@ do
     if ! grep -q -F "$i" "$rootdir"/ci/cppcheck/cppcheck_existing_issues.txt ; then
         status=1
         err "New issue:"
-        grep -F "$i" "$rootdir/cppcheck_results.txt"
+        "$i"
     fi
 done
 
+# Generate a new list of existing issues by removing the ones that are now solved
+comm -12 <(sort "$rootdir"/cppcheck_results.txt) <(sort "$rootdir"/ci/cppcheck/cppcheck_existing_issues.txt)  > "$rootdir"/cppcheck_existing_issues.txt
```

Comment:
This relies very heavily on the order of the cppcheck_results.txt file to be stable. We run cppcheck with just the rootdir argument. I don't know how it iterates, but I assume it uses directory order without sorting. In that case, the order is not stable, so we may get different results on different runs.

Reply:
The list of issues is always sorted first, so it should always produce the same result. Or am I misunderstanding something?